automation/travisci-test.sh

```diff
@@ -8,11 +8,7 @@ if [ -z "$TRAVIS_TAG" ]; then
 	while sleep 9m; do echo "Long running job - keep alive"; done &
 	LOOP_PID=$!
 
-	if [[ $TRAVIS_REPO_SLUG == "kubevirt/kubevirt" && $TRAVIS_CPU_ARCH == "amd64" ]]; then
-		make goveralls
-	else
-		make bazel-test
-	fi
+    make bazel-test
```

Comment:
do we want to run this?

Reply:
We should run this at least as long as the prow job `pull-kubevirt-unit-test` is not yet reporting to GitHub I think.